### DISCUSSION TITLE
Y+ calculation

### DISCUSSION THREAD

**Discussion by parisipa** [nextfoam/baram#97] [Q&A]
Hello all,

Has anyone found a way to calculate the y+ of your mesh in BaramFlow?

**Comment by kdg5424** [comment]
Hello @parisipa.
This is Donggyu from NEXTFOAM. Thank you for your interest in BARAM.

We have the plan for monitoring y+. We also know that it's important to monitor y+ for turbulence users.
So could you just wait until we implement it?

Thanks

Regards

Donggyu kim

**Comment by parisipa** [comment]
Hi all,

In case anyone encounters this thread with the same question, you can use the following solution:

echo "functions { yPlus1
{
    // Mandatory entries (unmodifiable)
    type            yPlus;
    libs            (fieldFunctionObjects);

    // Optional (inherited) entries
    writePrecision  8;
    writeToFile     true;
    useUserTime     true;
    region          region0;
    enabled         true;
    log             true;
    timeStart       0;
    timeEnd         1000;
    executeControl  timeStep;
    executeInterval 1;
    writeControl    timeStep;
    writeInterval   1;
}}" >> system/controlDict


When you calculation runs, open a linux terminal (or WSL) and get into the case directory. Use this command to manually add the y+ function to the controlDict while your simulation runs.

**Reply by kdg5424** [reply]
Thank you very much.

**Comment by jiban** [comment]
Hi @parisipa,
You can now calculate *Wall y plus* in BARAM Flow like following.

![image](https://github.com/user-attachments/assets/987e861c-9890-46e6-8f3d-e5a2cadb3297)

Thank you.

-- Jake
